book.rb

```diff
@@ -1,9 +1,9 @@
 class Book
   attr_accessor :title, :author, :rentals
 
-  def initialise(title, _author)
+  def initialise(title, author)
```

Comment:
- [ ] Hello, @lukman155. Excellent work on the changes suggested by the previous reviewer 👏. However, the error that occurs when we attempt to create a book or a rental has not been resolved. However, after going through your code to try to resolve the issue, I discovered that the error was caused by misspelling `initialize` in your `Book` and `Rental` 
classes. It is written with a `z` rather than an `s`. Please correct that misspelling, and creating a `Book` or a `Rental` will work.👌😊